### PR TITLE
style: separate shell commands

### PR DESCRIPTION
**Pull request by Rylan12**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

This is the main PR to update the style for `system` and `Utils.popen` commands to separate shell command arguments (see https://github.com/Homebrew/brew/pull/7691).

Example:

```ruby
system "foo bar --baz"
```

should become

```ruby
system "foo", "bar", "--baz"
```

I have tested all of these locally without any issues, but the CI will likely take a long time to complete (there are 46 formulae changed here)

Just FYI: there will be two additional PRs that deal with formulae that needed more significant changes or had build/test/audit issues

### CODE REVIEW

File: Formula/gauge.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class Gauge < Formula
   desc "Test automation tool that supports executable documentation"
-  homepage "https://getgauge.io"
+  homepage "https://gauge.org"
```
Comment (Rylan12) [inline Formula/gauge.rb:3]: The gauge homepage seems to have changed to https://gauge.org, so I went ahead and updated the homepage as well in this commit.

This is the only change in this PR that isn't separating shell command arguments, so I figured I'd point it out so it isn't missed.

**Review by chenrui333 — COMMENTED** [review]
Can this be also checked with `brew audit` as well??

PR lgtm

**Comment by Rylan12** [comment]
> Can this be also checked with `brew audit` as well??

Sorry I wasn't clear: https://github.com/Homebrew/brew/pull/7691 is adding a style check for this. This PR is just a precursor so that homebrew-core follows the new style guidelines.

**Comment by Rylan12** [comment]
pjproject is the only formula to fail in the CI. Here's the message:

```
==> Downloading from https://codeload.github.com/pjsip/pjproject/tar.gz/2.10
/usr/bin/curl --disable --globoff --show-error --user-agent Homebrew/2.4.2-35-gb4cd99c\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 10.15.5\)\ curl/7.64.1 --fail --silent --retry 3 --location --remote-time --continue-at 0 --output /Users/brew/Library/Caches/Homebrew/downloads/f6a80f8e2f5729cf8692ae983f2b19c763ef9f73ad009ecd9dc1e85902118f6a--tar.gz.incomplete https://codeload.github.com/pjsip/pjproject/tar.gz/2.10
curl: (22) The requested URL returned error: 500 Internal Server Error
Error: Failed to download resource "pjproject"
Download failed: https://github.com/pjsip/pjproject/archive/2.10.tar.gz
```

It works fine locally.

**Comment by Rylan12** [comment]
@chenrui333 is there something I can do for the CI error? I can't reproduce the problem locally. I'm thinking I might just need to rerun the CI.

If so, what time is best to do it? (expecting about a 6 hour run time)

**Comment by chenrui333** [comment]
> pjproject is the only formula to fail in the CI. Here's the message:
> 
> ```
> ==> Downloading from https://codeload.github.com/pjsip/pjproject/tar.gz/2.10
> /usr/bin/curl --disable --globoff --show-error --user-agent Homebrew/2.4.2-35-gb4cd99c\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 10.15.5\)\ curl/7.64.1 --fail --silent --retry 3 --location --remote-time --continue-at 0 --output /Users/brew/Library/Caches/Homebrew/downloads/f6a80f8e2f5729cf8692ae983f2b19c763ef9f73ad009ecd9dc1e85902118f6a--tar.gz.incomplete https://codeload.github.com/pjsip/pjproject/tar.gz/2.10
> curl: (22) The requested URL returned error: 500 Internal Server Error
> Error: Failed to download resource "pjproject"
> Download failed: https://github.com/pjsip/pjproject/archive/2.10.tar.gz
> ```
> 
> It works fine locally.

That looks right to me as well (not sure what is going on.)

**Comment by chenrui333** [comment]
I will test our this PR tomorrow, going to hit the bed now.

**Comment by Rylan12** [comment]
@chenrui333 have you been able to take a look at this?

**Comment by chenrui333** [comment]
> @chenrui333 have you been able to take a look at this?

Almost forgot, checking on your branch now.

**Comment by Rylan12** [comment]
Well, `pjproject` worked this time. `mmctl` failed but it looks like it just needs a rebase (`mmctl` was updated after I rebased I think)

@chenrui333 let me know what the best time is for me to run the CI again.

**Comment by glyng1** [comment]
Guys I am unable to complete the application process I need help.

I am also having problems with the phone connection to my laptop.

I cannot even get logged on and google account needs security checks.

So sorry to bother you.

I am a novice

**Comment by SMillerDev** [comment]
Just check if there are no long builds, otherwise you're good to re-run.

**Comment by chenrui333** [comment]
yeah, I think we dont have long builds now, feel free to do the rerun.

**Comment by Rylan12** [comment]
Done 🚀

**Comment by Rylan12** [comment]
Checks have passed! Not sure what was going on with `pjproject` before but it seems to be fine now.

**Review by chenrui333 — APPROVED** [review]
Nice work. 🚀

**Comment by Rylan12** [comment]
Thanks! If either of you wants to check out the companion PR (#57066) I'd appreciate it!

It's making the same style change as this one but on formulae with existing issues

**Pull request closed without merging**